test/jdk/java/lang/Thread/virtual/ThreadPollOnYield.java

```diff
@@ -36,7 +36,7 @@
  * @summary Test that Thread.yield loop polls for safepoints
  * @requires vm.continuations & vm.compMode != "Xcomp"
  * @library /test/lib
- * @run junit/othervm -Xcomp -XX:-TieredCompilation -XX:CompileCommand=inline,*::yield* -XX:CompileCommand=inline,*::*Yield ThreadPollOnYield
+ * @run junit/othervm/native --enable-native-access=ALL-UNNAMED -Xcomp -XX:-TieredCompilation -XX:CompileCommand=inline,*::yield* -XX:CompileCommand=inline,*::*Yield ThreadPollOnYield
```

Comment:
Would you mind splitting this line (jtreg is okay with that) as it's nearly 190 chars now so getting too long.